escalation/policy.go

```diff
@@ -23,3 +24,8 @@ func (p Policy) Normalize() (*Policy, error) {
 
 	return &p, nil
 }
+
+// IsUserFavorite returns a boolean value based on if the escalation policy is a user favorite
```

Comment:
nit: I know this text was lifted from the other ones, but we can probably make it a little more succinct.

```suggestion
// IsUserFavorite returns true if this policy is a favorite of the current user.
```

Reply:
addressed